lib/galaxy/webapps/tool_shed/app.py

```diff
@@ -10,9 +10,11 @@
 from galaxy.openid.providers import OpenIDProviders
 from galaxy.util.dbkeys import GenomeBuilds
 from galaxy.web import security
+from galaxy.config import configure_logging
```

Comment:
yeah, I had a look at the sorting we are supposed to use and still do not understand it :D